api/downloads/download-file-util.js

```diff
@@ -24,6 +24,7 @@ const errors = {
   CHUNK_SIZE_ERROR: "CHUNK_SIZE_ERROR",
   FILE_CREATING_ERROR: "FILE_CREATING_ERROR",
   FILE_WRITING_ERROR: "FILE_WRITING_ERROR",
+  NO_SPACE_LEFT_ERROR: "ENOSPC",
```

Comment:
Please use same convention like in the other errors for value 'NO_SPACE_LEFT_ERROR'.

Reply:
ENOSPC is the offical node.js error. It is not an error from downstream. Using the 'convention', I would have to add a transposition function, which is not necessary and efficient

Reply:
But you already do this here: https://github.com/castlabs/downstream_electron/pull/54/files#diff-9e4fcd4cb87f2c825ad90e1fa50cfdbfR87

Wouldn't it be just a matter of changing 
NO_SPACE_LEFT_ERROR: "ENOSPC"

to

NO_SPACE_LEFT_ERROR: "NO_SPACE_LEFT_ERROR",

and have more readable downstream error?

Reply:
Ok, but then il will have to change in my application
It is just a value. ENOSPC is an official error.